modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CProductLocalServiceImpl.java

```diff
@@ -74,7 +74,7 @@ public CProduct deleteCProduct(CProduct cProduct) throws PortalException {
 
 		// Commerce product definitions
 
-		_cpDefinitionLocalService.deleteCPDefinitions(
+		_cpDefinitionPersistence.removeByC_S(
```

Comment:
this I'm not sure will work, with the service call we delete all the related entities of the prodct, this way only the DB entities will be removed.

Reply:
@andrea-ale-sbarra Is that PR going to be merged soon?

Reply:
@andrea-ale-sbarra Ah I just see the meesage in the slack channel, thanks!

Reply:
Hi @andrea-ale-sbarra 

The commit https://github.com/liferay-commerce/liferay-portal/pull/2997/commits/769758030a90ad035215fc49869e94417192eab3 of the pull:
https://github.com/liferay-commerce/liferay-portal/pull/2997 has removed all the usage of CPDefinitionLocalService in CProductLocalServiceImpl, but still kept the reference of CPDefinitionLocalService in it, why should we keep it?

@dantewang @tinatian  if the reference of CPDefinitionLocalService is removed in CProductLocalServiceImpl in the [COMMERCE](https://github.com/liferay-commerce/liferay-portal/pull/2997)  change, then we do not need to do the change of LPS-161562 in this pull.

Lily